libvast/src/detail/mmapbuf.cpp

```diff
@@ -105,6 +105,15 @@ bool mmapbuf::resize(size_t new_size) {
   // Resize the underlying file, if available.
   if (fd_ != -1 && ftruncate(fd_, new_size) < 0)
     return false;
+#if __linux__
+  int flags = MREMAP_MAYMOVE;
+  auto* map = mremap(map_, size_, new_size, flags);
```

Comment:
`auto map` (the `*` is redundant).

Reply:
In this case you are right, but there is a subtle nuisance of c++s auto type deduction rules that we have to keep in mind: if you want to assign the the returned pointer to a `const` variable, you have to use `const auto*`. `const auto` would result in `T* const` instead. For this reason, I prefer using `auto*` and `const auto*` instead of `auto` and `const auto*`.

I will change the line here nonetheless to comply with the coding style, but I wanted to mention the issue.